ClinGen/clincoded
# Update to Parse VCV-Formatted Data for Interpretations Submitted to ClinVar

Steps to test #2143:
1. Start a new variant curation.
2. Retrieve and save the variant to be taken to the evidence view. Once there, confirm that the **Overall ClinVar Interpretation** and **Interpretations Submitted to ClinVar (Germline SCVs only)** panels have data.

Note: The data displayed in the VCI should closely match what's available at ClinVar (the top "summary" and **Submitted interpretations** sections, respectively).

For ClinVar ID 139214 (images from https://www.ncbi.nlm.nih.gov/clinvar/variation/139214/):
![image](https://user-images.githubusercontent.com/32915557/76868627-6e7cb780-6824-11ea-8d8f-26f4322f5c7c.png)

![image](https://user-images.githubusercontent.com/32915557/76868546-54db7000-6824-11ea-8281-cb9ca77ee05a.png)
